frontend/src/store/actions/Review/ReviewActions.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable max-len */
```

Comment:
For now, I think this file shouldn't be included in this PR.

Reply:
I will delete this file. Thanks